SCons/Util.py

```diff
@@ -99,7 +99,7 @@ def splitext(path) -> tuple:
     sep = rightmost_separator(path, os.sep)
     dot = path.rfind('.')
     # An ext is only real if it has at least one non-digit char
```

Comment:
It's kind of a silly claim, anyway (yes,I know this isn't part of the change)... even on Windows, file extensions can have digits in them. Consider `.mp4`...`

Reply:
What's wrong? this PR doesn't change functionality of splitext(), it's just simplify implementation.
```python3
print(splitext("film.mp4")) #  ('film', '.mp4')
print(splitext("hell.666")) #  ('hell.666', '')

Reply:
nothing wrong with the change - anyway, I misread the comment.